Pod/Classes/URBNDataSourceAdapter.h

```diff
@@ -9,16 +9,18 @@
 #import <Foundation/Foundation.h>
 #import <UIKit/UIKit.h>
 
+NS_ASSUME_NONNULL_BEGIN
+
 typedef NS_ENUM(NSUInteger, URBNSupplementaryViewType) {
     URBNSupplementaryViewTypeHeader,
     URBNSupplementaryViewTypeFooter
 };
 
-typedef NSString *(^URBNSupplementaryViewReuseIdentifierBlock) (NSString *kind, NSIndexPath *indexPath);
-typedef NSString *(^URBNReuseableIdentifierBlock) (id item, NSIndexPath *indexPath);
+typedef NSString * _Nonnull (^URBNSupplementaryViewReuseIdentifierBlock) (NSString *kind, NSIndexPath * _Nullable indexPath);
```

Comment:
Ugh, nope. For some reason I though `[NSIndexPath indexPathForItem:0 inSection:section]` could return nil, but look at that beautiful `_Nonnull`:
![screen shot 2015-11-02 at 4 22 11 pm](https://cloud.githubusercontent.com/assets/71474/10894642/ecc75bc8-817d-11e5-8a45-441e458ebe5f.png)